src/sst/core/main.cc

```diff
@@ -628,7 +631,7 @@ main(int argc, char *argv[])
             delete your_graph;
         }
 
-        if ( *my_ranks.begin() != myRank.rank) cout << "ERROR" << endl;
```

Comment:
Should this actually cause an fatal(), rather than just printing "ERROR"... with no indication of what kind of error is going on?

Reply:
Probably.  Looks like I just changed the cout to printf (wasn't sure if Output was initialized here), but didn't take a good look at what it was doing.  I'm not actually sure what error that's checking for.  I'll look at the code a bit deeper and either add a reasonable error message or remove the check if it's not needed.  It has the feel of a temporary debug printout, but it's been in there for so long, I don't remember for sure.

Reply:
This was an old debug statement and has been removed entirely.